reimbursement.md

```diff
@@ -4,17 +4,16 @@ Fidel Reimbursement is an add-on product to _Transaction_ tracking capability, t
 
 ## Availability
 
-Reimbursement supports both debit and credit cards by Visa and Mastercard in the US. Your account must be set up in the US to have access to the Reimbursement product.
-
+Reimbursement supports both debit and credit cards by Visa and Mastercard in the US or UK. Your account must be set up either in the US or the UK to have access to the Reimbursement product.
 ## Activation
 
-If your account is set up in the US and you are using an older [API version](https://reference.fidel.uk/docs/version-log) (reimbursement requires version `2021-09-28` or later) or haven't agreed to the _North America_ [Terms & Conditions](https://fidel.uk/legal) (_17 Jun 2021_), you will be prompted to go through the Reimbursement product activation.
+If your account is set up in the US or UK and you are using an older API version (reimbursement requires version 2021-09-28 or later) or haven't agreed to the [Terms & Conditions](https://fidel.uk/legal) in your region, you will be prompted to go through the Reimbursement product activation.
```

Comment:
Shouldn't we use the new URL for the T&C link? `https://fidelapi.com/legal`

Reply:
Unrelated to this PR, but do we need to update the T&Cs on the legal page?

Reply:
you're right, i'll update this

Reply:
that's a good point - we can bring this into Ruben's attention